src/Translator.php

```diff
@@ -298,6 +298,25 @@ private function mergeYaml(): void
                 $contentYaml = $this->arrayMergeRecursive($contentYaml, $parsedYaml);
                 $this->recursiveKeySort($contentYaml);
             }
+            // Remove any keys where the value is the same as the source english value
+            $enPath = preg_replace('#/[a-zA-Z_]+\.yml#', '/en.yml', $path);
```

Comment:
This regex is too restrictive. The CMS module for example has `sr_RS@latin.yml`.
By this point we already know this is the path to a lang file so we don't have to prescribe the file name format.
```suggestion
            $enPath = preg_replace('#/[^/\.]+\.yml$#', '/en.yml', $path);
```

Reply:
Updated